### PR TITLE
add unit tests to alerts, values, and route to increase test coverage

### PR DESCRIPTION
**Pull request by tariq1890**
Adding some unit tests

### CODE REVIEW

File: model/alert_test.go [hunk 1]
```diff
@@ -116,3 +118,160 @@ func TestAlertValidate(t *testing.T) {
 		}
 	}
 }
+
+func TestAlert(t *testing.T) {
+	// test unresolved alert
```
Comment (brian-brazil) [inline model/alert_test.go]: Comments should be full sentences, starting with a capital letter and ending with a full stop.

Comment (tariq1890) [reply model/alert_test.go]: Sure!

Comment (tariq1890) [reply model/alert_test.go]: @brian-brazil Fixed

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**